test/src/primitives/src/value/H256.ts

```diff
@@ -6,7 +6,7 @@ const RLP = require("rlp");
 export type H256Value = H256 | string;
 
 /**
- * Handles 256-bit data. Used to express block hash, parcel hash, transaction hash, merkle root, etc.
```

Comment:
What is a parcel hash?

Reply:
Parcel is a collection of transactions. They are not used anymore.

Reply:
ok